README.md

```diff
@@ -55,16 +55,16 @@ view.backgroundColor = [UIColor orangeColor];
 UIApplication.sharedApplication.delegate;
 ```
 
-Dot-notation should **not** be used to call methods.
+Bracket notation should be used to call methods.
 
-**For example:**
+**Do this:**
 ```objc
-array.firstObject
+[array firstObject]
```

Comment:
This example is incorrect because firstObject is a property.
A better example would be `[set anyObject]`